esx_service/vmci/connection_types.h

```diff
@@ -29,6 +29,13 @@
 #define START_NON_PRIVILEGED_PORT 1024
 
 /*
+ * Check and set errno helper
+ * Useful when send/recv gets us less than we wanted, abd we want to set errno
```

Comment:
Ok. For e.g. if create is partially successful i.e. ESX has created a disk but if we retry CREATE, does ESX returns success?